wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController+Scrolling.swift

```diff
@@ -52,11 +52,22 @@ extension ConversationContentViewController {
         updateTableViewHeaderView()
     }
 
-    func scrollToBottom() {
-        guard !isScrolledToBottom else { return }
+    /// Scrolls the tableView to the bottom-most row.
+    ///
+    /// This method checks if the tableView is not already scrolled to the bottom.
+    func scrollToBottomIfNeeded() {
+        let maxMessageCountForAnimation = 20
+
+        guard !isScrolledToBottom else {
+            return
+        }
 
         dataSource.loadMessages()
-        tableView.scroll(toIndex: 0)
+
+        let lastRowIndexPath = IndexPath(row: 0, section: 0)
+        let shouldAnimate = !UIAccessibility.isReduceMotionEnabled && dataSource.messages.count <= maxMessageCountForAnimation
+
+        tableView.scrollToRow(at: lastRowIndexPath, at: .bottom, animated: shouldAnimate)
```

Comment:
Why don't you want to use same `tableView.scroll(toIndex: 0)` but with an additional `animated` parameter?

Reply:
It is a custom method and we don't have the ability to animate using tableView.scroll(toIndex: 0). There is no additional parameter: animated.

Reply:
You can add this parameter and it has a check that John mentioned above.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/f1ff30b49e7c016d6056822b8d4376f3522cfbc6